pkg/common/configs/configvalidator.go

```diff
@@ -139,6 +139,72 @@ func checkQueueResource(cur QueueConfig, parentM *resources.Resource) (*resource
 	return curG, nil
 }
 
+func checkLimitResource(cur QueueConfig, parent *QueueConfig, users map[string]*resources.Resource, groups map[string]*resources.Resource) error {
```

Comment:
I am thinking, if we can storage the hierarchy queue limit value to memory when traversing, so that we can use it in UGM, so that we don't need to traverse the value again when we want to storage the limit value?

Reply:
After traversing, when there is no err, we can return the user limit and group limit all value to store.